docs/docs/cmd/spo/roledefinition/roledefinition-get.md

```diff
@@ -0,0 +1,27 @@
+# spo roledefinition get
+
+Gets specified role definition from web by id
+
+## Usage
+
+```sh
+m365 spo roledefinition get [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+: URL of the site for which to retrieve role definition
+
+`-i, --id <id>`
+: role definition id
```

Comment:
Let's start the sentence with uppercase `R`

Reply:
✅